### PR TITLE
Update contextual_bandits.md

### PR DESCRIPTION
**Pull request by mikedropmedia**
updated cb copy

### CODE REVIEW

File: _guides/contextual_bandits.md [hunk 1]
```diff
@@ -10,106 +10,110 @@ body_class: tutorial
 resource_icon: /svg/resources/guide.svg
 ---
 
-# Contextual Bandits in VW
+# Contextual Bandits and Vowpal Wabbit
```
Comment (pavithraks) [inline _guides/contextual_bandits.md]: no "test" data, only "data". test data means something else in machine learning parlance.

**Comment by cheng-tan** [comment]
@pavithraks have you finished reviewing this PR? Has the changes for MAB section got in?

@mikedropmedia new changes removed some of the citations, which needs to be added back. Basically anywhere with {% cite .... %} needs to be added to the new version.

File: _guides/contextual_bandits.md [hunk 2]
```diff
@@ -9,106 +9,112 @@ layout: tutorial
...
-### What is a Contextual Bandit?
 
-Consider an application that interacts with its environment, such as a news website with users or a cloud controller with machines. Let's call this application _APP_. This application repeatedly goes through the following:
-1. Some context x arrives and is observed by _APP_
-2. _APP_ chooses an action _a_ from a set of actions _A_ i.e. a ∈ A to take (A may depend on x)
-3. Some reward r for the chosen a is observed by _APP_
+## Getting started with Vowpal Wabbit
 
-We want our application _APP_ to take actions such that we get the highest possible reward. In machine learning parlance, we want a _model_ that tells us which action to take.
+To install VW—and for more information on building VW from source or using a package manager—see [Getting started](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Getting-started) guide.
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]:
```suggestion
To install VW—and for more information on building VW from source or using a package manager—see [Getting started](getting_started.html) guide. 
```

File: _guides/contextual_bandits.md [hunk 3]
```diff
@@ -9,106 +9,112 @@ layout: tutorial
...
+## Getting started with Vowpal Wabbit
 
-We want our application _APP_ to take actions such that we get the highest possible reward. In machine learning parlance, we want a _model_ that tells us which action to take.
+To install VW—and for more information on building VW from source or using a package manager—see [Getting started](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Getting-started) guide. 
 
-This scenario is very similar to a traditional <a href="https://en.wikipedia.org/wiki/Multi-armed_bandit" target="_blank">Multi Armed Bandit (MAB)</a>. The term "multi-armed bandit" comes from a hypothetical experiment where a gambler must choose between multiple actions (i.e. slot machines, the "one-armed bandits"), each with an unknown payout. The goal is to maximize the payout by optimally choosing the best actions when odds and payouts are unknown.
+>**Note:** The CB tutorial uses [Vowpal Wabbit Python package](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python). Additional binary packages are available for select platforms. See [Getting started](https://cheng-tan.github.io/vowpalwabbit.github.io/) module on the homepage for more information.
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]:
```suggestion
>**Note:** The CB tutorial uses [Vowpal Wabbit Python package](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python). Additional binary packages are available for select platforms. See [Getting started](getting_started.html) module on the homepage for more information.
```

File: _guides/contextual_bandits.md [hunk 4]
```diff
@@ -9,106 +9,112 @@ layout: tutorial
 tags: contextual&nbsp;bandits
 ---
 
-## Contextual Bandits in VW
+# Contextual Bandits and Vowpal Wabbit
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]: The header hierarchical starts from 'h2' in tutorial. Basically, all h1 needs to be h2, h2 needs to be h3 etc

File: _guides/contextual_bandits.md [hunk 5]
```diff
@@ -9,106 +9,112 @@ layout: tutorial
...
 - Inverse Propensity Score <sup>{% cite doi:10.1080/01621459.1952.10483446 %}</sup> `--cb_type ips`
 - Doubly Robust <sup>{% cite DBLP:conf/icml/JiangL16 %} {% cite DBLP:conf/icml/DudikLL11 %}</sup> `--cb_type dr`
 - Direct Method `--cb_type dm`
 - Multi Task Regression/Importance Weighted Regression <sup>{% cite bietti2018a %} {% cite Karampatziakis:2011:OIW:3020548.3020594 %}</sup> `--cb_type mtr`
 
-For more details, please refer to the <a href="https://arxiv.org/pdf/1802.04064.pdf" target="_blank">Contextual Bandit bake-off paper</a>.
+There are multiple policy evaluation approaches available to optimize a policy. VW offers four approaches to specify CB approach using `--cb_type`:
 
-**CB algorithms** in VW can be classified as:
+- **Inverse Propensity Score:** `--cb_type ips`
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]: is this section duplicate with line 63 to line 66?

File: _guides/contextual_bandits.md [hunk 6]
```diff
@@ -9,106 +9,112 @@ layout: tutorial
...
+VW offers five exploration algorithms:
 
-- Explore-first `--first`
-- Epsilon-greedy `--epsilon`
-- Bagging Explorer <sup>{% cite DBLP:journals/corr/OsbandR15 %} {%cite DBLP:journals/corr/EcklesK14 %}</sup> `--bag`
-- Online Cover <sup>{% cite DBLP:journals/corr/AgarwalHKLLS14 %}</sup> `--cover`
-- Softmax Explorer <sup>{% cite DBLP:journals/corr/abs-1811-04383 %}</sup> `--softmax` (only supported for `--cb_explore_adf`)
+- **Explore-First** `--first`
+- **Epsilon-Greedy** `--epsilon`
+- **Bagging Explorer** `--bag`
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]: Adding the citations here?

File: _guides/contextual_bandits.md [hunk 7]
```diff
@@ -9,106 +9,112 @@ layout: tutorial
...
+- **Online Cover** `--cover`
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]: citation

File: _guides/contextual_bandits.md [hunk 8]
```diff
@@ -9,106 +9,112 @@ layout: tutorial
...
+- **Softmax Explorer** `--softmax` (only supported for `--cb_explore_adf`)
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]: citation

File: _guides/contextual_bandits.md (hunk 6 above)
Comment (mikedropmedia) [reply _guides/contextual_bandits.md]: Ok, where can I find the original citations? I don't have them on any of my versions

Comment (cheng-tan) [reply _guides/contextual_bandits.md]: if you view the file changes in split view, looking at the code on the left pane. Searching for "bagging explorer", {% cite .... %} is the original citations.

File: _guides/contextual_bandits.md [hunk 9]
```diff
@@ -9,106 +9,106 @@ layout: tutorial
...
 ---
 
-## Contextual Bandits in VW
+## Contextual bandits and Vowpal Wabbit
 
-The goal of this tutorial is to have you walk away with an understanding of Contextual Bandits, when CB can be used, how to run CB algorithms in Vowpal Wabbit (VW), and hopefully make you feel empowered and excited to use it on your own. This tutorial focuses on Python but VW is also supported in C++ and C#.
+This tutorial includes an overview of the contextual bandits approach to reinforcement learning and describes how to approach a contextual bandit problem using Vowpal Wabbit. You will learn how to use Vowpal Wabbit in a contextual bandit setting with the Python tutorial—including when and how to work with different contextual bandits approaches, how to format data, and understand the results. No prior knowledge of contextual bandits or Vowpal Wabbit is required.
 
-### What is a Contextual Bandit?
+## Getting started with Vowpal Wabbit
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]: should these headers be h3? Only the tutorial title is h2?

**Pull request merged**